avocado/utils/asset.py

```diff
@@ -84,7 +84,7 @@ def fetch(self):
         if self.nameobj.scheme:
             urls.append(self.nameobj.geturl())
 
-        # First let's find for the file in all cache locations
+        # First let's search for the file in each one of the cache locations
```

Comment:
I'm not native, but `search` is definitely better fitting here. Or `look`. As for `each one of the` again, not native, but how about `in every cache location`? Anyway this version is also fine by me, it just sounds strange to my Czech ears.

Reply:
It sounds fine to me...